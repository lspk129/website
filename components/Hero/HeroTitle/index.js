import styled from 'styled-components';

const HeroTitle = styled.h1`
  text-align: center;
  position: absolute;
  top: 35%;
  left: 50%;
  transform: translate(-50%, -50%);
  color: white;
  font-size: 4rem;
  white-space: nowrap;
  font-weight: bold;
  text-shadow: 0 0.1rem 0.1rem rgba(0, 0, 0, 0.4);
  line-height: 4rem;
`;

export default HeroTitle;
