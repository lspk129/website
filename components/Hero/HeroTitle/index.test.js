import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import HeroTitle from './index';

describe('HeroTitle', () => {
  it('renders correctly', () => {
    const wrapper = renderer.create(<HeroTitle />);
    const tree = wrapper.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders h1 element', () => {
    const wrapper = shallow(<HeroTitle />);
    expect(wrapper).toBePresent();
    expect(wrapper).toHaveTagName('h1');
  });

  it('renders text "Moka Haiku:web app agency"', () => {
    const wrapper = shallow(<HeroTitle>Moka Haiku:web app agency</HeroTitle>);
    expect(wrapper).toHaveText('Moka Haiku:web app agency');
  });
});
