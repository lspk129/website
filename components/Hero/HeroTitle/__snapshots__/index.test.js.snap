// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroTitle renders correctly 1`] = `
<h1
  className="sc-bdVaJa drKbDu"
/>
`;
